spring-cloud-gcp-autoconfigure/src/test/java/com/google/cloud/spring/autoconfigure/pubsub/health/PubSubHealthIndicatorAutoConfigurationTests.java

```diff
@@ -46,37 +64,131 @@
 			.withBean(GcpProjectIdProvider.class,  () -> () -> "fake project")
 			.withBean(CredentialsProvider.class, () -> () -> mock(Credentials.class));
 
+	@SuppressWarnings("unchecked")
 	@Test
-	public void healthIndicatorPresent() {
+	public void healthIndicatorPresent() throws Exception {
+		PubSubTemplate mockPubSubTemplate = mock(PubSubTemplate.class);
+		ListenableFuture<List<AcknowledgeablePubsubMessage>> future = mock(ListenableFuture.class);
```

Comment:
Don't need to mock it; use `ApiFutures.immediateFuture()`. It might help with unchecked warnings, too, since you can return a typed list.

Reply:
Since I actually want to verify calls on the future, I think I'll keep the mock.